components/org.wso2.carbon.identity.oauth.par/src/main/java/org/wso2/carbon/identity/oauth/par/core/OAuthParRequestWrapper.java

```diff
@@ -51,10 +56,22 @@ public OAuthParRequestWrapper(HttpServletRequest request, Map<String, String> pa
     @Override
     public String getParameter(String name) {
 
+        // Remove param request_uri to avoid conflicting with OIDC requests passed by reference.
+        if (StringUtils.equals(name, OAuthConstants.OAuth20Params.REQUEST_URI)) {
```

Comment:
Shall we simply check like OAuthConstants.OAuth20Params.REQUEST_URI.equals(name) here. Csn remove the dependency to StringUtils